app/views/expenses/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<section class="expense-show">
+  <nav class="common-nav">
+    <%= link_to "←", expenses_path, class: "back-home" %>
+    <div class="page-name">Payments</div>
```

Comment:
- [ ] @Al-Amin-Khan-Shakil Great work implementing the changes requested, however, you still have not changed the page titles as previously requested, On this page instead of `Payments` it should be  `TRANSACTIONS`, kindly consider implementing these changes requested.